tests/testsuite/init.rs

```diff
@@ -657,3 +657,46 @@ fn no_filename() {
         )
         .run();
 }
+
+#[cargo_test]
+fn formats_source() {
+    fs::write(&paths::root().join("rustfmt.toml"), "tab_spaces = 2").unwrap();
```

Comment:
Oh, I'm sorry, I forgot one other thing. This should check that rustfmt is installed and return if it is not.  Just change [this function](https://github.com/rust-lang/cargo/blob/acf88cc26ab935a9f652612c4c5739a8e94c6686/crates/cargo-test-support/src/lib.rs#L1804-L1812) to take a process by name so that we can reuse it.  (Rename it `process_is_available` or something like that.)

Reply:
Yay, I get to refactor something!

Will do tomorrow :)

Would `command_is_available` be better? `process` seems to refer to the OS process that will be started as a result.

Reply:
Yea, sounds good to me!

Reply:
Hmm, I have second thoughts about changing the function. Using it with the third command will probably require changing the function once again. Maybe I should just copy-paste it and name it `rustfmt_not_available`...

Reply:
I wouldn't worry about it.  If we need to change it later, that's easy to do.